README.md

```diff
@@ -61,10 +61,25 @@ See [Troubleshooting](website/docs/troubleshooting.html.markdown).
 ## Developing the Provider
 See [CONTRIBUTING.md](./CONTRIBUTING.md).
 
-## Autoclose stale issues and PRs
+## Issues
+
+### Autoclose stale issues and PRs
+- After 5 days of no activity (no comments or commits on an issue/PR) we automatically tag it as "stale" and add a message: ```This issue/PR has gone 7 days without any activity and meets the project's definition of "stale". This will be auto-closed if there is no new activity over the next 7 days. If the issue is still relevant and active, you can simply comment with a "bump" to keep it open, or add the label "not_stale". Thanks for keeping our repository healthy!```
```

Comment:
The number of days is not consistent, is it 5 or 7?

Reply:
5, I will update it , thanks!